### PR TITLE
Vumc GitHub publishing

### PR DESCRIPTION
**Pull request by NickofNasa**
Added VUMC github specific publishing block to the API service, this will allow publishing to the VUMC jar repo once a token is provided to the databiosphere repository.

### CODE REVIEW

File: service/build.gradle [hunk 1]
```diff
@@ -105,6 +105,9 @@ dependencies {
         exclude group: "org.junit.vintage", module: "junit-vintage-engine"
     }
 
+    // Cloud SQL
+    implementation "com.google.cloud.sql:postgres-socket-factory:1.7.0"
```
Comment (marikomedlock) [inline service/build.gradle:109]: Could you pull this dependency out into a separate PR please? I want to make sure we do some testing on our end before merging, and don't want to block the publishing change you made.

Comment (NickofNasa) [reply service/build.gradle]: removed this implementation from this pull request

**Review by melissachang — APPROVED** [review]
